src/components/HelFormFields/HelLabeledCheckboxGroup.js

```diff
@@ -1,106 +1,83 @@
+import './HelLabeledCheckboxGroup.scss'
+
 import PropTypes from 'prop-types';
-import React from 'react'
-import PureRenderMixin from 'react-addons-pure-render-mixin'
+import React, {useRef} from 'react'
 import {Checkbox} from 'react-bootstrap'
-
-import {map, isEqual} from 'lodash'
-
 import {connect} from 'react-redux'
-import {setData} from '../../actions/editor'
-
+import {setData as setDataAction} from '../../actions/editor'
 import ValidationPopover from '../ValidationPopover'
 
-// NOTE: Not using ES6 classes because of the needed mixins
-class HelLabeledCheckboxGroup extends React.Component {
-    constructor(props) {
-        super(props)
-
-        this.handleChange = this.handleChange.bind(this)
-    }
-    
-    handleChange() {
-        const {options} = this.props
-
-        let checked = options.reduce((ac, op, index) => {
-            if(this[`checkRef${index}`].checked) {
-                ac.push(this[`checkRef${index}`]) 
+const HelLabeledCheckboxGroup = (props) => {
+    const {options, name, selectedValues, itemClassName, groupLabel, validationErrors} = props
```

Comment:
You can also destructure the props at the top of the function:

```js
const HelLabeledCheckboxGroup = ({options, name, selectedValues, itemClassName, groupLabel, validationErrors}) => ...
```

Reply:
👍